src/containers/LearningResourcePage/components/MetaImageInformation.jsx

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Burde jeg kanskje gjøre denne mer generisk slik at visuelt element også kan bruke den? Ble litt usikker så hadde satt pris på innspill her. @oyvinmar @sebastianjg

Reply:
Eventuelt hva kan felles tittel for disse være?

Reply:
Bare MetaInformation?